legend-depot-core-services/src/main/java/org/finos/legend/depot/services/projects/ProjectsServiceImpl.java

```diff
@@ -355,4 +355,15 @@ private StoreProjectVersionData getProject(String groupId, String artifactId, St
         return projectData.get();
     }
 
+    private List<ProjectVersion> overrideDependencies(List<ProjectVersion> directDependencies, List<ProjectVersion> projectDependencies)
```

Comment:
this functionality should be in the dependencies manager ..where all the rest of deps logic si